symengine/cwrapper.cpp

```diff
@@ -1832,6 +1834,28 @@ CWRAPPER_OUTPUT_TYPE basic_as_numer_denom(basic numer, basic denom,
     CWRAPPER_END
 }
 
+CWRAPPER_OUTPUT_TYPE basic_add_as_two_terms(basic term1, basic term2, const basic s)
+{
+    CWRAPPER_BEGIN
+    SYMENGINE_ASSERT(is_a<Add>(*(s->m)));
+    if (is_a<Add>(*(s->m))) {
+        rcp_static_cast<const Add>(s->m)->as_two_terms(
+            SymEngine::outArg(term1->m), SymEngine::outArg(term2->m));
+    }
```

Comment:
remove the "if", since this is checked by the assert now.

Reply:
Oops sorry, I missed that completely